tests/fv3_conf/cplhist_cdeps_docn_dice.IN

```diff
@@ -0,0 +1,8 @@
+mkdir -p INPUT
+
+#CDEPS data files
+#rsync -arv @[INPUTDATA_ROOT]/cplhist_DOCN_DICE/. INPUT
+cp /scratch1/NCEPDEV/stmp2/Nick.Szapiro/FV3_RT/rt_301994/cpld_control_nowave_noaero_p8_intel/ufs.cpld.cpl*.nc INPUT
```

Comment:
I think it makes more sense to ncrcat these various files into single files rather than having 120 ice files and 24 ocean files coupler history files.

Reply:
Only 2 ice and 2 ocean files are used in the actual regression test. I will clean this up for the final input data directory. The syntax for the input filelist follows a hafs test and is a bit tricky (see stream_files_docn and stream_files_dice in tests/tests/atm_ds2s_docn_dice) if it's worth keeping the example